.ci/openshift_e2e.sh

```diff
@@ -58,6 +65,50 @@ function getDevWorkspaceOperatorLogs() {
     oc get events -n ${NAMESPACE}| tee get_events.log
 }
 
+function generateHtpasswd() {
+   htpasswd -c -B -b ${PATH_TO_HTPASSWD_FILE} ${TERMINAL_USER_LOGIN} ${TERMINAL_USER_PASSWORD}
+}
+function generateHtpasswdProviderYaml() {
```

Comment:
nitpick: newline

Reply:
It was removed after implementing of solution with Service Account